rosserial_vex_cortex/src/rosserial_vex_cortex/SerialClient.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
Can we not use this file from another instance/location? I don't see any local modifications.